test/martha/forceAccessUrl.test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('ava');
+const sinon = require('sinon');
+const config = require('../../common/config');
+const helpers = require('./_helpers.js')
+
+const { marthaV3Handler: marthaV3 } = require('../../martha/martha_v3');
+const apiAdapter = require('../../common/api_adapter');
+
+const terraAuth = "Bearer XYZ_TerraAuthToken";
+const bucketId = "bucket_id";
+const responseId = `v1_ResponseId_${bucketId}`;
+const gsAccessId = "gcp-us-central1";
+const gsUrl = `gs://broad-jade-dev-data-bucket/BucketIdPrefix/${bucketId}`;
+const drsUri = `drs://${config.HOST_TDR_DEV}/${responseId}`
+
+
+const jadeAccessUrlMetadataResponse = {
```

Comment:
in the original test suite, this object is [defined in an all-encompassing "resources" file](https://github.com/broadinstitute/martha/blob/dev/test/martha/_martha_v3_resources.js#L163).

In this test, it appears to need far fewer fields than provided in that resources file. The definition here demonstrates the essential fields.

Is this an example of DRY gone too far? We may have to repeat versions of this object in other tests, but keeping it small/constrained in each test makes it easier to read.